### ISSUE TITLE
Legibility!

### ISSUE THREAD

**Issue by austinbeer** [nltk/nltk_book#169]
Hi. I'm just a designer poking around and I see it's super hard to read this page!!! Here's really simple css you could use that dramatically makes it easier to read. Didn't spend too much time on it, but you get the idea - bigger, narrower text.

.document {

    font-size: 1.4em;
    line-height: 1.6em;
    
}
h1 {
      height: 55px;
      padding-top: 55px; 
        
    }
p , ul , h1 {
    width: 50%;
    padding-left:25%;
}

**Comment by gregjd** [comment]
I agree. My first reaction when I opened a chapter was, "Wait, is it just me, or is this page really wide?" It definitely makes it harder to read.
